.buildkite/scripts/publish_air_gapped_docs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eo pipefail
+
+# Log in to docker.elastic.co.
+set +x
+DOCKER_USERNAME=$(vault kv get -field=username secret/ci/elastic-docs/docker.elastic.co)
+DOCKER_PASSWORD=$(vault kv get -field=password secret/ci/elastic-docs/docker.elastic.co)
+source .buildkite/scripts/retry.sh
+retry 5 docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD docker.elastic.co
+unset DOCKER_USERNAME
+unset DOCKER_PASSWORD
+set -x
+
+cd "$(git rev-parse --show-toplevel)"
+source ./air_gapped/build_bk.sh
+docker push "$AIR_GAPPED"
```

Comment:
I know this was as in in the Jenkins world, but it's weird to push to $AIR_GAPPED where we don't see where it was sourced from. Maybe we should add AIR_GAPPED as either passed in or falling back on a default path? and same in the build script. If we actually do that, we can override the registry path in the top level job directly (in the pipeline.yml file) - that'll altogether be more readable and cleaner.

Reply:
i can move it to pipeline yaml it is make sense, but i don't think that we need to override registry path using ui. Does it make sense? just put export to pipeline.yaml file

Reply:
I was thinking of doing something like: 

```
AIR_GAPPED_REGISTRY_PATH="${AIR_GAPPED_REGISTRY_PATH:=docs-private/air_gapped}"
AIR_GAPPED=docker.elastic.co/$AIR_GAPPED_REGISTRY_PATH:latest 
```
in both the build.sh and publish.sh script, and simply adding the below in the pipeline.yaml

```
env:
   AIR_GAPPED_REGISTRY_PATH=docs-private/bk_air_gapped
```

Reply:
hmmm, i don't know, looks too much for me, maybe do something like:

Reply:
The intent is to allow folks to run the script locally without having it throw an error - but I guess we wouldn't want them to push to the production path anyway.

Reply:
okay, let it be